config/src/config/quorum_store_config.rs

```diff
@@ -29,7 +29,7 @@ impl Default for QuorumStoreBackPressureConfig {
         QuorumStoreBackPressureConfig {
             // QS will be backpressured if the remaining total txns is more than this number
             // Roughly, target TPS * commit latency seconds
-            backlog_txn_limit_count: 8000 * 2,
+            backlog_txn_limit_count: 12_000,
```

Comment:
should we make this dependent on the block size, or at least have a comment to refer at the block size that this needs to be changed too?